Formula/swift-format.rb

```diff
@@ -31,10 +31,6 @@ class SwiftFormat < Formula
   uses_from_macos "swift"
 
   def install
-    # Support current stable Swift.
-    # Remove with Swift 5.7.1.
-    inreplace "Package.swift", '.upToNextMinor(from: "0.50700.0")', '.exact("0.50700.0")' if OS.mac?
-
     # This can likely be removed with 0.50800.0
```

Comment:
```suggestion
     # Support Swift 5.7.
     # Remove when minimum supported Swift >= 5.7.1.
     inreplace "Package.swift", '.upToNextMinor(from: "0.50700.0")', '.exact("0.50700.0")' if OS.mac? && build.stable?
     
    # This can likely be removed with 0.50800.0
```

guessing at @cho-m 's suggestion